### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by bastjan**
## Checklist

- [x] PR contains a single logical change (to build a better changelog).
- [x] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.

### CODE REVIEW

File: class/defaults.yml [hunk 1]
```diff
@@ -1,4 +1,133 @@
...
+        persistentVolume:
+          size: 50Gi
+
+      distributor:
+        replicas: 3
+
+      ingester:
+        persistentVolume:
+          size: 50Gi
+        replicas: 4
```
Comment (simu) [inline class/defaults.yml:73]: Is the ingester intentionally configured with 4 replicas by default?

File: docs/modules/ROOT/pages/references/parameters.adoc [hunk 2]
```diff
@@ -2,7 +2,11 @@
 
 The parent key for all of the following parameters is `mimir`.
 
-== `namespace`
+The component is allows multi instantiation.
```
Comment (simu) [inline docs/modules/ROOT/pages/references/parameters.adoc]:
```suggestion
The component allows multi instantiation.
```

**Review by simu — APPROVED** [review]
LGTM overall, I did not look at the rendered Mimir helm chart in detail.

**Comment by simu** [comment]
On second thought: the component could probably benefit from some additional documentation which explains why the default configuration provided in parameter `helm_values` were chosen.

**Comment by bastjan** [comment]
I'll try to add some documentation for the `helm_values`.

**Comment by simu** [comment]
> I'll try to add some documentation for the `helm_values`.

It doesn't need to be detailed, but a short explanation (or link to best practices or recommendations) for the default number of replicas for each component would be nice to have.

File: class/defaults.yml (hunk 1 above)
Comment (bastjan) [reply class/defaults.yml:73]: AFAIK it is recommended by Grafana to prevent data/quorum loss. I'll search for the documentation

Comment (bastjan) [reply class/defaults.yml:73]: https://grafana.com/docs/mimir/latest/operators-guide/architecture/components/ingester/#replication

https://github.com/grafana/mimir/blob/a13959db5d38ff65c2b7ef52c56331d2f4dbc00c/operations/helm/charts/mimir-distributed/small.yaml#L53

**Comment by bastjan** [comment]
I added the most important links in the last commit

**Review by glrf — APPROVED** [review]

**Review by simu — APPROVED** [review]

**Pull request merged**